scripts/makeportable.sh

```diff
@@ -67,45 +69,57 @@ cp "${ROOT_DIR}/resources/streamlink-script.py" "${bundle_dir}/streamlink-script
 cp "${ROOT_DIR}/resources/streamlink.bat" "${bundle_dir}/streamlink.bat"
 cp "${ROOT_DIR}/NOTICE" "${bundle_dir}/NOTICE.txt"
 
-cp -r "${STREAMLINK_CHECKOUT_DIR}/win32/config" "${bundle_dir}/config.template"
+cp -r "${STREAMLINK_WINDOWS_INSTALLER_CHECKOUT_DIR}/files/config" "${bundle_dir}/config.template"
 
 ASSETS_DATA=$(cat "${STREAMLINK_ASSETS_FILE}")
 
 assets_prepare() {
     log "Preparing assets"
-    while read -r filename sha256 url; do
-        if ! [[ -f "${cache_dir}/${filename}" ]]; then
-            log "Downloading asset: ${filename}"
-            curl -L -o "${cache_dir}/${filename}" "${url}"
-        fi
-        echo "${sha256} ${cache_dir}/${filename}" | sha256sum --check -
-    done < <(jq -r '.[] | "\(.filename) \(.sha256) \(.url)"' <<< "${ASSETS_DATA}")
+    if [[ "${STREAMLINK_PYTHON_ARCH}" == "amd64" ]]; then
```

Comment:
I'm not very familiar with bash, so I don't know how to do string var replacement when piping a command into a while loop. I tried making it use a quoted string, but it wasn't able to resolve the command like that.

This also doesn't handle non-ffmpeg assets, which are theoretically possible, but not currently in the config. I'm definitely not familiar enough with bash to write this generically.